### ISSUE TITLE
Causes an error on WAVE accessibility report

### ISSUE THREAD

**Issue by dobe0002** [sagalbot/vue-select#1072]
**Describe the bug**
When a vue-select items is viewed in the browser and a WAVE report is run, an accessibility issue is flagged. 

This is important because our institution (along with many others) determine the accessibility status of an application/page based on the results of a WAVE report.

WAVE is indicating that the "Clear selection" button doesn't have an accessible name or containing text, so a screen reader user cannot read or access the button.

**To Reproduce**
1. Install [WAVE Browser Plugin](https://wave.webaim.org/extension/)
2. Go to a page in a browser that has the vue-select like the [vue-select homepage](https://vue-select.org/)
3. Run WAVE report
4. Look in WAVE results under "Errors"

**Expected behavior**
When WAVE is run, no errors are indicated

**Screenshots**
See attached
<img width="1175" alt="Screen Shot 2020-02-27 at 3 37 29 PM" src="https://user-images.githubusercontent.com/13108611/75489597-2a5e6b80-5978-11ea-887d-04b5df9f6475.png">

**Desktop (please complete the following information):**
 - macOS (10.14.6)
 - Chrome (but may be an issue in others)
 - Version 80.0.3987.122

**Comment by sagalbot** [comment]
Thanks for pointing me to this tool, very helpful. Fixed in the `v3.5.2` release.
